mix.exs

```diff
@@ -39,7 +39,7 @@ defmodule TelemetryMetricsStatsd.MixProject do
   defp deps do
     [
       {:telemetry, "~> 0.4 or ~> 1.0"},
-      {:telemetry_metrics, "~> 0.6"},
+      {:telemetry_metrics, "~> 1.0"},
```

Comment:
We should still accept 0.6, not doing this would be a breaking change for _this_ library. We can bump requirement to 1.0 when we release 1.0 of telemetry metrics statsd.

Reply:
understood. that totally make sense. 
how about using a "or" here? so current repo could be compatible with 1.0 telemetry_metrics